python/tvm/relay/op/strategy/arm_cpu.py

```diff
@@ -250,6 +257,7 @@ def conv2d_strategy_arm_cpu(attrs, inputs, out_type, target):
                         wrap_compute_conv2d(topi.arm_cpu.conv2d_nhwc_spatial_pack),
                         wrap_topi_schedule(topi.arm_cpu.schedule_conv2d_nhwc_spatial_pack),
                         name="conv2d_nhwc_spatial_pack.arm_cpu",
+                        plevel=5,
```

Comment:
The default priority level of an implementation is 10. When two or more implementations are added to the strategy, in the untuned case, the one with the highest priority is chosen. Also, if two implementations have the same plevel, the one added to the strategy first is selected.   

So actually, the change doesn't change the strategy selection behaviour at all, since `spatial_pack` is added last in all cases and every schedule has the default priority level, so it would already only get used when no other implementation was added.  

I'm not sure if having an explicit plevel makes this more or less understandable. I am happy to remove it.

Reply:
Thanks for the explanation, I think it makes sense to keep the change, to prevent that schedule accidentally becoming priority schedule in some future refactor of that incredibly confusing monster if else 😅